newTab.html

```diff
@@ -1,143 +1,110 @@
+                    <label class="checkbox"></label>
+                    <label>Include suggested sites</label>
+                </div>
+            </div>
+            <div id="newtab-customize-blank" class="newtab-customize-panel-item selectable">
+                <label>Show blank page</label>
+            </div>
+            <div id="newtab-customize-learn" class="newtab-customize-panel-item">
+                <label>Learn about New Tab</label>
             </div>
-        </div>
-        <div id="newtab-customize-blank" class="newtab-customize-panel-item selectable">
-            <label>Show blank page</label>
-        </div>
-        <div id="newtab-customize-learn" class="newtab-customize-panel-item">
-            <label>Learn about New Tab</label>
         </div>
     </div>
-  </div>
-
-  <div id="newtab-customize-overlay"></div>
-
-  <div id="newtab-intro-mask">
-    <div id="newtab-intro-modal">
-
-      <div id="newtab-intro-progress">
-        <div id="newtab-intro-numerical-progress"></div>
-        <div id="newtab-intro-graphical-progress"></div>
-          <span id="indicator"/>
+    <div id="newtab-customize-overlay"></div>
+    <div id="newtab-intro-mask">
+        <div id="newtab-intro-modal">
+            <div id="newtab-intro-progress">
+                <div id="newtab-intro-numerical-progress"></div>
+                <div id="newtab-intro-graphical-progress"></div>
+                <span id="indicator"></span>
+            </div>
         </div>
-      </div>
-      <div id="newtab-intro-header"/>
-      <div id="newtab-intro-body">
-        <div id="newtab-intro-content">
-          <div id="newtab-intro-image"></div>
-          <div id="newtab-intro-text">
-            <p/><p/>
-          </div>
+        <div id="newtab-intro-header"></div>
+        <div id="newtab-intro-body">
+            <div id="newtab-intro-content">
+                <div id="newtab-intro-image"></div>
+                <div id="newtab-intro-text">
+                    <p></p>
+                </div>
+            </div>
+            <div id="newtab-intro-buttons">
+                <button onclick="gIntro._back()"></button>
+                <button onclick="gIntro._next()"></button>
+            </div>
         </div>
-        <div id="newtab-intro-buttons">
-          <input type="button" onclick="gIntro._back()"/>
-          <input type="button" default="true" onclick="gIntro._next()"/>
+        <div id="newtab-intro-footer">
+            <ul>
+                <li></li>
+                <li></li>
+            </ul>
         </div>
-      </div>
-      <div id="newtab-intro-footer">
-        <ul>
-          <li/><li/>
-        </ul>
-      </div>
     </div>
-  </div>
-
-  <div id="newtab-scrollbox">
-
-    <div id="newtab-vertical-margin">
-
-      <div id="newtab-margin-top"></div>
-
-      <div id="newtab-margin-undo-container">
-        <div id="newtab-undo-container" undo-disabled="true">
```

Comment:
In undo.js we add and remove `undo-disabled` and have specific CSS for it so switching it to `data-undo-disabled` breaks some of the functionality